m4/ax_boost_base.m4

```diff
@@ -33,7 +33,7 @@
 #   and this notice are preserved. This file is offered as-is, without any
 #   warranty.
 
-#serial 47
+#serial 49
```

Comment:
Why skip 48?

Reply:
The serial number of each macros is the number of commits that modified it in the repository.

Reply:
Oh, thanks!